_posts/en/newsletters/2021-04-07-newsletter.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Bitcoin Optech Newsletter #143'
+permalink: /en/newsletters/2021/04/07/
+name: 2021-04-07-newsletter
+slug: 2021-04-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains our regular sections with announcements
+of new releases and release candidates, plus notable changes to popular
+Bitcoin infrastructure projects.
+
+## News
+
+*No noteworthy news to report this week.*
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+
+- [C-Lightning 0.10.0][C-Lightning 0.10.0] is the newest major release
+  of this LN node software. It contains a number of enhancements to its
+  API and includes experimental support for [dual funding][topic dual
+  funding].
+
+- [BTCPay 1.0.7.2][] fixes a couple minor issues discovered after last
+  week's security release.
+
+<!-- FIXME:harding to update tuesday -->
```

Comment:
maybe https://github.com/bisq-network/bisq/releases/tag/v1.6.2

<details><summary>summary</summary><p>

```
v1.6.2

@ripcurlx ripcurlx released this 2 days ago · 89 commits to master since this release

This is a hotfix release that fixes an issue with SPV resync introduced in v1.6.0 and fixes a minor bug in the portfolio view when price nodes aren't available.

For more details please see https://github.com/bisq-network/bisq/milestone/56?closed=1

Here are the release notes from v1.6.0:
Release notes

This release ships the official beta release for the Bisq API! It has been tested on mainnet, but is still in beta, so please be careful.

Other highlights: absolute minimum deposits have now been lowered from 0.006 BTC to 0.001 BTC. Maker and taker fee transactions are now verified via mempool.space to reduce trade issues, and mediation and arbitration were improved as well. And as always, there are many bug fixes and refinements.

Also important: this release ships part of 1 of 2 for BSQ SegWit implementation. It prepares the network for handling SegWit BSQ transactions (activation block is 680300, around 2021/04/25 -- all full nodes should update to 1.6.0+ before that time). Shortly after this activation point, the wallet code (#5109) will be merged into master for testing so that the full BSQ SegWit update is shipped in the following release.
```
</p></details>